repos/system_upgrade/el7toel8/actors/networkmanagerupdateclientid/actor.py

```diff
@@ -28,7 +26,7 @@ def process(self):
                 return
 
             try:
-                r = call(['nm-update-client-ids.py'])
+                r = run(['nm-update-client-ids.py'])['stdout']
```

Comment:
note for me: missing `split=True`? check the context later

Reply:
hey, thanks for pointing that out. You are right, my PR is changing existing behaviour and I will fix it. I did not get that during tests because the result of this call is only used in logging messages. Thanks for the other pair of eyes.